sdk/spring-sdk/src/it/java/com/example/wiring/EventSourcedEntityIntegrationTest.java

```diff
@@ -110,7 +110,12 @@ public void verifyCounterEventSourcedAfterRestartFromSnapshot() {
     restartCounterEntity(counterId);
 
     // current state is based on snapshot and should be the same as previously
-    Assertions.assertEquals(10, getCounter(counterId));
+    await()
+      .ignoreExceptions()
+      .atMost(10, TimeUnit.of(SECONDS))
+      .until(
+        () ->getCounter(counterId),
+        new IsEqual(10));
```

Comment:
Here we call it after restarting the entity. I'm assuming here that it simply took too long to restart it so using `await.util` to retry it.